std/fs/write_json_test.ts

```diff
@@ -33,14 +22,11 @@ Deno.test("writeJsonIfExists", async function (): Promise<void> {
 
   await Deno.writeFile(existsJsonFile, new Uint8Array());
 
-  await assertThrowsAsync(
-    async (): Promise<void> => {
-      await writeJson(existsJsonFile, { a: "1" });
-      throw new Error("should write success");
-    },
-    Error,
-    "should write success",
-  );
+  try {
+    await writeJson(existsJsonFile, { a: "1" });
+  } catch {
+    // empty
+  }
```

Comment:
Do we need empty catches? Doesn't Deno.test fail automatically if there is an uncaught error?